.travis.yml

```diff
@@ -1,66 +1,21 @@
 sudo: required
 
+language: c
+
 services:
     - docker
-    
+
 cache: ccache
```

Comment:
Now that all the stages except for the coverity one are gone, I figure `ccache` added in https://github.com/systemd/systemd/pull/9067 could be deleted.